.circleci/config.yml

```diff
@@ -2,21 +2,42 @@ version: 2
 jobs:
   build:
     docker:
-      - image: python:3.7-alpine
+      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:tap-tester-v4
     steps:
       - checkout
       - run:
           name: 'Setup virtual env'
           command: |
-            mkdir -p ~/.virtualenvs
-            apk add build-base libffi-dev
-            pip install virtualenv
-            python -m virtualenv ~/.virtualenvs/tap-slack
-            source ~/.virtualenvs/tap-slack/bin/activate
+            python3 -mvenv /usr/local/share/virtualenvs/tap-slack
+            source /usr/local/share/virtualenvs/tap-slack/bin/activate
             pip install -U 'pip<19.2' setuptools
             pip install .[dev]
+      - add_ssh_keys
       - run:
-          name: 'pylint'
+          name: 'Integration Tests'
           command: |
-            source ~/.virtualenvs/tap-slack/bin/activate
-            pylint tap_slack -d C,R,W,no-member
```

Comment:
Does the below still run `pylint` or should we also add that back as another step?

Reply:
In the stitch environment I believe plyint is still run without this command.